publish-alpha.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
You will want to add `set -euo pipefail` here.

Reply:
this is just designed to be run locally, it's not part of this (or any orb's) config